src/react-app-env.d.ts

```diff
@@ -1,2 +1,8 @@
 // eslint-disable-next-line
 /// <reference types="react-scripts" />
+
+type Tab = {
```

Comment:
```suggestion
type TabInterface = {
```
название пересекается с названием компонента